zero/src/prover_state/cli.rs

```diff
@@ -62,6 +62,9 @@ macro_rules! gen_prover_state_config {
             pub persistence: CircuitPersistence,
             #[clap(long, help_heading = HEADING, default_value_t = TableLoadStrategy::OnDemand)]
             pub load_strategy: TableLoadStrategy,
+            /// Run in an unsafe but fast configuration for testing
```

Comment:
Can you explain what `unsafe` means here? What's the tradeoff to the user? What does it mean if it passes?

Reply:
`unsafe` may not be the appropriate term here. I've updated the comments to reflect that. Passing tests with `use_test_config` confirms the logic and correctness of the code, but it doesn't guarantee the security of the proofs generated, as this configuration sacrifices security for speed.

Reply:
I don't really understand what security means in this context, but I'm happy with the change

Reply:
Our current STARK configuration targets a 100-bit security level, meaning an attacker would need to try 2^100 times to create a fraudulent proof that the verifier would accept. However, we can lower this security level in exchange for faster performance during testing.

Reply:
@0xaatif are you good with this part, or should Sai reword the one-liner?